Sources/Widgets/Widgets3D/PolyLineWidget/behavior.js

```diff
@@ -83,16 +83,25 @@ export default function widgetBehavior(publicAPI, model) {
     ) {
       return macro.VOID;
     }
-
-    if (model.activeState === model.widgetState.getMoveHandle()) {
+    const manipulator =
+      model.activeState?.getManipulator?.() ?? model.manipulator;
+    if (
+      model.activeState === model.widgetState.getMoveHandle() &&
+      manipulator
+    ) {
       updateMoveHandle(e);
+      const worldCoords = manipulator.handleEvent(
+        e,
+        model._apiSpecificRenderWindow
+      );
       // Commit handle to location
       const moveHandle = model.widgetState.getMoveHandle();
+      moveHandle.setOrigin(worldCoords);
       const newHandle = model.widgetState.addHandle();
-      newHandle.setOrigin(...moveHandle.getOrigin());
+      newHandle.setOrigin(moveHandle.getOrigin());
       newHandle.setColor(moveHandle.getColor());
       newHandle.setScale1(moveHandle.getScale1());
-      newHandle.setManipulator(model.manipulator);
+      newHandle.setManipulator(manipulator);
```

Comment:
@tomsuchel WDYT ?

Reply:
LGTM: it seems logic to use the `activeState` (=moveHandle) manipulator in priority to set the `newHandle` manipulator